dependencies.yaml

```diff
@@ -92,6 +100,17 @@ channels:
   - conda-forge
   - nvidia
 dependencies:
+  rapids_build_backend:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - rapids-build-backend>=0.3.0,<0.4.0.dev0
+      - output_types: [requirements, pyproject]
+        packages:
+          - scikit-build-core[pyproject]>=0.7.0
+      - output_types: [conda]
+        packages:
+          - scikit-build-core>=0.7.0
```

Comment:
Match the order used in [other repos](https://github.com/rapidsai/cudf/blob/dec0354b1ac2af981d4e8f13aceb45365838a1d8/dependencies.yaml#L296).
```suggestion
      - output_types: [conda]
        packages:
          - scikit-build-core>=0.7.0
      - output_types: [requirements, pyproject]
        packages:
          - scikit-build-core[pyproject]>=0.7.0
```